### PR TITLE
Added newssection html and both css

### PR DESCRIPTION
**Pull request by itsashishbharti**
dnl
mnl

### CODE REVIEW

File: blogs/01/styles/newsletter-section.css [hunk 1]
```diff
@@ -1,79 +1,73 @@
...
-{
-	border:#1b1bc3;
+#newsletter > .subscribe > .container > .box  > div.ipbox{
+	height:43px;
+	width:470px;
+	display:flex;
+}
+#newsletter > .subscribe > .container > .box  > div.ipbox > input.submitbtn {
+	height:43px;
+	width:98.16px;
```
Comment (srikant96) [inline blogs/01/styles/newsletter-section.css:39]: Fix width

File: blogs/01/styles/newsletter-section.css [hunk 2]
```diff
@@ -1,79 +1,73 @@
...
+	background-color:var(--span-blue);
+	border:none;
+}
+
+#newsletter > .subscribe > .container > .box  > div.ipbox > input.inptbox{
+	height:39px;
+	width:371.84px;
```
Comment (srikant96) [inline blogs/01/styles/newsletter-section.css:46]: remove .84 from `width`

**Review by srikant96 — CHANGES_REQUESTED** [review]
MTC

**Comment by amezng** [comment]
not as per the design

**Pull request closed without merging**